### PR TITLE
Improve documentation some more

### PR DESCRIPTION
**Pull request by spetterman66**
I hope this helps a bit :)

### CODE REVIEW

File: docs/archlinux.md [hunk 1]
```diff
@@ -23,15 +23,15 @@ qemu-img create arch.img 10G
...
 
-For keyboard support it is necessary to open /etc/mkinitcpio.conf and edit the following line:
+For keyboard support, it is necessary to open /etc/mkinitcpio.conf and edit the following line:
 
 ```sh
 MODULES="atkbd i8042"
 ```
 
-For the changes to take effect you need to regenerate the RAMdisk with `mkinitcpio -p linux`
+For the changes to take effect, you need to regenerate the tempoary file system with `mkinitcpio -p linux`
```
Comment (cyberplant) [inline docs/archlinux.md:32]: Hello! there's a typo in 'tempoary'.

File: docs/archlinux.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+The last ISO image installer version of Arch Linux that supports the 32-bit architecture is version [2017.02.01](https://www.archlinux.org/releng/releases/2017.02.01/). Later versions of the Arch Linux ISO images don't work on the v86 emulator, because the installer only supports the 64-bit architecture. For existing Arch Linux installations, updates and patches will be done until somewhere around 2018.
 
-In the future the community might come up with an alternative distribution based on Archlinux to maintain support for x86. At this point in time [archlinux32](https://mirror.archlinux32.org) seems to work.
+In the future, the community might come up with an alternative distribution based on Arch Linux to maintain support for the 32-bit architecture. At this point in time [archlinux32](https://mirror.archlinux32.org) seems to work.
```
Comment (copy) [inline docs/archlinux.md:8]: It'd probably better to delete the first paragraph, and refer to archlinux32 directly.

File: docs/cpu-idling.md [hunk 3]
```diff
@@ -11,20 +10,17 @@ Here some solutions for different OSes:
...
 1. Run `edit C:\fdconfig.sys` (or `edit C:\config.sys`)
 2. Add to file: `IDLEHALT=1`
 3. Save changes (*press Alt + F and x*) and restart FreeDOS.
 
-
 ## Windows 9x (using AmnHLT)
 1. Download `amnhltm.zip` from http://toogam.com/software/archive/drivers/cpu/cpuidle/amnhltm.zip ([mirror](https://web.archive.org/web/20060212132151/http://www.user.cityline.ru/~maxamn/amnhltm.zip))
-2. Unzip archive in any location.
-3. **Note**: If you have installed VBE9x, restart Windows, press F8 on boot, select *Command prompt only*, run `cd C:\path\to\amnhlt\` and follow to next step.
+2. Unzip the compressed archive in any location.
```
Comment (copy) [inline docs/cpu-idling.md:20]:
```suggestion
2. Unzip the archive in any location.
```

File: docs/filesystem.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-A 9p filesystem is supported by the emulator, using a virtio transport. Using
+A 9p file system is supported by v86, using a virtio transport. Using
```
Comment (copy) [inline docs/filesystem.md:1]: "filesystem" is correct (see https://en.wikipedia.org/wiki/File_system), and more consistent with the rest of the docs.

**Comment by copy** [comment]
Thanks!

**Pull request merged**